lib/routes/transfer_funds_in_progress_dialog.dart

```diff
@@ -29,15 +29,26 @@ class _TransferFundsInProgressDialog extends StatefulWidget {
 class _TransferFundsInProgressDialogState
     extends State<_TransferFundsInProgressDialog> {
   StreamSubscription<AccountModel> _stateSubscription;
+  ModalRoute _currentRoute;
 
   @override
   void initState() {
     super.initState();
     _stateSubscription = widget.accountStream.listen((state) {
-      if (state.transferringOnChainDeposit != true) {
-        _pop();
+      if (state.transferringOnChainDeposit != true && _currentRoute.isActive) {
+        Navigator.of(context).removeRoute(_currentRoute);
```

Comment:
I don't  think you can access build context in initState.

Reply:
These are not accessed in initState per se but after subscription to a stream has established, we're also checking if _currentRoute.isActive beforehand. It's not really needed on this example but what we can do to make things more robust is scheduling the stream subscription logic once the context is available as such:
```
    WidgetsBinding.instance.addPostFrameCallback((_) {
      /// Stream Subscription Logic
    });
```

Reply:
Why do we wish to do this in the `initstate` instead of calling this in `didChangeDependencies`

Reply:
We want to subscribe to the stream only once and `didChangeDependencies` may be called multiple times per widget lifecycle. We'd need some preventative measures in `didChangeDependencies`, we do have some examples using `_isInit` variable on breezmobile.

Reply:
Got it, thanks for the clarifying.